DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Browser/Browser.aspx.cs

```diff
@@ -2419,10 +2419,20 @@ private void UploadFile(HttpPostedFile file, string command)
                 }
                 else
                 {
-                    var fileUrl = string.Format(!MapUrl(uploadPhysicalPath).EndsWith("/") ? "{0}/{1}" : "{0}{1}", MapUrl(uploadPhysicalPath), fileName);
-                    this.Response.ClearContent();
-                    this.Response.ContentType = "application/json";
-                    this.Response.Write($"{{\"uploaded\": 1, \"fileName\": \"{fileName}\", \"url\": \"{fileUrl}\"}}");
+                    // querystring parameter responseType equals "json" when the request comes from drag/drop
+                    if (this.Request.QueryString["responseType"]?.Equals("json", StringComparison.InvariantCultureIgnoreCase) ?? false)

```

Comment:
```suggestion
                    if (this.Request.QueryString["responseType"]?.Equals("json", StringComparison.InvariantCultureIgnoreCase) == true)
```

This is a little clearer for me to understand, do you think so?